common/src/main/resources/db/changelog/test_only/truncate-fake-data.sql

```diff
@@ -0,0 +1,9 @@
+--liquibase formatted sql
+--  -------------------------------------------------------------------------------------------------------------------
+
+--changeset earl_nolan:AB2D-2921 failOnError:true context:test
+truncate user_role;
+truncate user_account cascade;
+truncate role cascade;
+truncate contract cascade;
+
```

Comment:
If you want the index to reset to 1 for the tables, you can do this (for example):

TRUNCATE user_account RESTART IDENTITY CASCADE;

Reply:
Good to know.  I don't think it really matters for this case.